com.unity.render-pipelines.high-definition/Runtime/Lighting/AtmosphericScattering/Fog.cs

```diff
@@ -200,4 +218,71 @@ public sealed class FogColorParameter : VolumeParameter<FogColorMode>
         public FogColorParameter(FogColorMode value, bool overrideState = false)
             : base(value, overrideState) { }
     }
+
+    /// <summary>
+    /// This enum defines the two modes for controlling the quality and cost of the volumetric fog.
+    /// </summary>
+    public enum FogControl
+    {
+        /// <summary>
+        /// In this mode, the user changes the parameters on higher abstraction level centered around performance.
+        /// </summary>
+        Balance,
+
+        /// <summary>
+        /// In this mode, the user has a direct access to the internal parameters for the volumetric fog.
+        /// </summary>
+        Manual
```

Comment:
Use this mode if you want to have direct access to the internal properties that control volumetric fog.